packages/terra-application-links/src/tabs/ApplicationTabs.jsx

```diff
@@ -176,19 +176,24 @@ class ApplicationTabs extends React.Component {
         collapsedTabs.push(<CollapsedTab {...tabProps} />);
       }
     });
+    const tabClass = cx(
+      'tabs-container',
+      { 'is-calculating': this.isCalculating },
+      alignment,
+    );
 
     return (
-      <div {...customProps} className={cx(['tabs-wrapper'])}>
+      <div {...customProps} className={cx('tabs-wrapper')}>
```

Comment:
This should have applied  the theme context class name here

Reply:
Added 32feabb